fdbserver/tester.actor.cpp

```diff
@@ -376,7 +376,8 @@ ACTOR Future<Void> testDatabaseLiveness( Database cx, double databasePingDelay,
 	loop {
 		try {
 			state double start = now();
-			TraceEvent(("PingingDatabaseLiveness_" + context).c_str());
+			auto traceMsg = "PingingDatabaseLiveness_" + context;
```

Comment:
I'm not actually sure why clang 10 warns on the old code /shrug

Reply:
This one I didn't understand - but as it only happened once I figured this is not too bad to use a temporary for it...